src/quickpicks/schemaRegistries.ts

```diff
@@ -29,52 +32,93 @@ export async function schemaRegistryQuickPickWithViewProgress(): Promise<
- * of through the view->item->context menu.
+ * Create a quickpick to let the user choose a Schema Registry, named by environment.
  *
  * @returns The selected Schema Registry, or undefined if none was selected.
  */
 export async function schemaRegistryQuickPick(): Promise<SchemaRegistry | undefined> {
-  // TODO(shoup): update to support LocalSchemaRegistry
+  const localSchemaRegistries: LocalSchemaRegistry[] = [];
+  let cloudSchemaRegistries: CCloudSchemaRegistry[] = [];
 
   // schema registries are a coarse resource, so ensure they are loaded before proceeding
   const preloader = CCloudResourcePreloader.getInstance();
   await preloader.ensureCoarseResourcesLoaded();
 
-  // list all Schema Registries for all CCloud environments for the given connection; to be
-  // separated further by environment in the quickpick menu below
-  const resourceManager = getResourceManager();
-  const ccloudSchemaRegistries = Array.from(
-    (await resourceManager.getCCloudSchemaRegistries()).values(),
-  );
+  // first we grab all available (local+CCloud) Schema Registries
+  let localGroups: LocalResourceGroup[] = [];
+  let cloudGroups: CCloudSchemaRegistryByEnv | null = new Map<string, CCloudSchemaRegistry>();
+  [localGroups, cloudGroups] = await Promise.all([
+    getLocalResources(),
+    getResourceManager().getCCloudSchemaRegistries(),
+  ]);
+  localGroups.forEach((group) => {
+    if (group.schemaRegistry) {
+      localSchemaRegistries.push(group.schemaRegistry);
+    }
+  });
 
-  if (ccloudSchemaRegistries.length === 0) {
-    let message = "No Schema Registries available.";
+  // list all Schema Registries for all CCloud environments for the given connection; to be separated
+  // further by environment in the quickpick menu below
+  let cloudEnvironmentIds: string[] = [];
+  if (hasCCloudAuthSession()) {
+    cloudGroups.forEach((schemaRegistry, envId) => {
+      cloudEnvironmentIds.push(envId);
+      cloudSchemaRegistries.push(schemaRegistry);
+    });
+  }
 
+  let availableSchemaRegistries: SchemaRegistry[] = [];
+  availableSchemaRegistries.push(...localSchemaRegistries, ...cloudSchemaRegistries);
+  if (availableSchemaRegistries.length === 0) {
+    vscode.window.showInformationMessage("No Schema Registries available.");
     if (!hasCCloudAuthSession()) {
-      message += " Perhaps log into to Confluent Cloud first?";
-    }
-    const login = "Log in to Confluent Cloud";
-    const selected = await vscode.window.showInformationMessage(message, login);
-    if (selected === login) {
-      vscode.commands.executeCommand("confluent.connections.create");
+      const login = "Log in to Confluent Cloud";
+      vscode.window
+        .showInformationMessage("Connect to Confluent Cloud to access remote clusters.", login)
+        .then((selected) => {
+          if (selected === login) {
+            vscode.commands.executeCommand("confluent.connections.create");
+          }
+        });
```

Comment:
I think we could consolidate this logic (and similar logic in the Kafka cluster quickpick) to conditionally show the CCloud login button instead of showing two notifications.